### PR TITLE
docs(v2): add docs on static assets

### PR DESCRIPTION
**Pull request by endiliey**
## Motivation

part of #1640  Guides / Docs / Static Assets

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

netlify tbd
https://deploy-preview-1648--docusaurus-2.netlify.com/docs/static-assets

### CODE REVIEW

**Comment by endiliey** [comment]
test https://deploy-preview-1648--docusaurus-2.netlify.com/docs/static-assets

File: website/docs/static-assets.md [hunk 1]
```diff
@@ -3,15 +3,40 @@ id: static-assets
...
+JSX example:
+
+```jsx
+// reference static/img/slash-birth.png
+<img src="/img/slash-birth.png" alt="logo" />
+```
+
+Result:
+
+<img src="/img/slash-birth.png" alt="logod" />
```
Comment (wgao19) [inline website/docs/static-assets.md]: Can we use a more meaningful alt?

File: website/docs/static-assets.md [hunk 2]
```diff
@@ -3,15 +3,40 @@ id: static-assets
...
+
+
+Keep in mind that:
+
+- None of the files in static folder be post-processed or minified.
```
Comment (wgao19) [inline website/docs/static-assets.md]: ...folder _will_ be post-processed or minified.

File: website/docs/static-assets.md (hunk 1 above)
Comment (endiliey) [reply website/docs/static-assets.md]: feel free to patch if u dont mind

**Pull request merged**